celery/django-celery-beat
# Add Farsi translation

This PR introduces Farsi translation support, enhancing the accessibility and usability of the module for Persian-speaking users. The Farsi translation aligns with the commitment to making the module more inclusive and user-friendly for a wider audience.

### How to Test
1. Ensure the application is set to Farsi locale.
2. Verify that all texts and messages appear in Farsi when the application is viewed.
3. Confirm that the translations are accurate and contextually appropriate.

### Screenshots
![image](https://github.com/celery/django-celery-beat/assets/5591800/705e4fd3-b3a5-4f35-ac3c-1cab00ecf842)

### Closing Note
I would like to express my gratitude for the opportunity to contribute. If any further changes or improvements are needed, please feel free to let me know. I look forward to hearing your feedback.
